src/components/Table/GridTableApi.ts

```diff
@@ -26,36 +26,35 @@ export function useGridTableApi<R extends Kinded>(): GridTableApi<R> {
 /** Provides an imperative API for an application page to interact with the table. */
 export type GridTableApi<R extends Kinded> = {
   /** Scrolls row `index` into view; only supported with `as=virtual` and after a `useEffect`. */
-  scrollToIndex: (index: number) => void;
```

Comment:
Made all of these the same syntax, plus a little re-ordering, otherwise no change.